pkg/grpc/label_store/protos/label_store.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package label_store_protos;
+
+service LabelStore {
+  rpc WatchMatches (WatchMatchesRequest) returns (stream WatchMatchesResponse) {}
+}
+
+enum LabelType {
+  unknown = 0;
+  pod = 1;
+  node = 2;
+  pod_clusters = 3;
+  replication_controller = 4; // singular because these match consul trees :(
+  rolls = 5;
+}
+
+message WatchMatchesRequest {
+  string selector = 1;
+  LabelType label_type = 2;
+}
+
+message Labeled {
+  LabelType label_type = 1;
+  string id = 2;
+  map<string,string> labels = 3;
```

Comment:
protobuf maps can only use "integral types" for keys. values can be whatever we want, although you cannot do a typedef of a primitive type e.g. `message Value string`.

Then we sort of lose the enforcement that values have to be strings, and if we ever json marshal the type it'll be messed up. I don't anticipate us doing that though. Probably using <string, string> is acceptable and won't kill us, i do wish we could do typedefs though